fdbclient/CommitProxyInterface.h

```diff
@@ -123,8 +124,8 @@ struct ClientDBInfo {
 		if constexpr (!is_fb_function<Archive>) {
 			ASSERT(ar.protocolVersion().isValid());
 		}
-		serializer(ar, grvProxies, commitProxies, id, clientTxnInfoSampleRate, clientTxnInfoSizeLimit, forward,
-		           transactionTagSampleRate, transactionTagSampleCost);
+		serializer(ar, grvProxies, commitProxies, id, history, clientTxnInfoSampleRate, clientTxnInfoSizeLimit,
+	               forward, transactionTagSampleRate, transactionTagSampleCost);
```

Comment:
I don't think for this one it is particularly important, but it's a good practice to always append new field in the `serializer` call. That way your change will be protocol compatible.